### ISSUE TITLE
Suggestion: New placeholder %townyadvanced_player_location_group_name%

### ISSUE THREAD

**Issue by JonatanLaMerce** [TownyAdvanced/Towny#5539]
Hello everybody!

As I specify in the title, I suggest a new placeholder called %townyadvanced_player_location_group_name% which would serve to show the name of the group of a plot, currently %townyadvanced_player_location_plot_name% does not show anything if it is a group of plots, only if the plot has a name assigned to it.

Plot groups can be named, but as far as I can see, this placeholder does not take into account plot group names.

I don't know if there is a way to collect the name of a group of plots or it could be implemented.

Thanks a lot!

**Comment by LlmDl** [comment]
All plot groups should be sharing the same name. The existing plot_name placeholder should reflect the group name if it has one.

I would have thought the name is being set to all the plot group's plots.

**Comment by JonatanLaMerce** [comment]
as you can see in the images, I think it does not work correctly, in the groups of plots, you set a name, but when you set a name to the plot with /plot set name indicates that it is not possible because it belongs to a group of plots.

I have tested the placeholder with denizen and deluxe menus, and in both the name appears empty if the plot belongs to a group of plots.

![2021-12-17 13_34_49-Minecraft 1 16 5 - Multijugador (servidor de terceros)](https://user-images.githubusercontent.com/16003674/146545749-a171d37a-5ff0-464b-8785-a446cb92f821.png)
![2021_12_17_13_34_00_Minecraft_1 16 5_Multijugador_servidor_de_terceros_](https://user-images.githubusercontent.com/16003674/146545754-738eec69-f088-498a-a873-768acfcee714.png)
![2021-12-17 13_32_51-Minecraft 1 16 5 - Multijugador (servidor de terceros)](https://user-images.githubusercontent.com/16003674/146545756-e5255275-faf2-409a-9b0b-ebbf7400a9ea.png)
![2021-12-17 13_35_30-](https://user-images.githubusercontent.com/16003674/146545758-9be6b1eb-16dd-405a-bce4-ec24e5b5794f.png)

**Comment by LlmDl** [comment]
```
  - New Placeholder: %townyadvanced_player_location_plotgroup_name%
    - Shows the PlotGroup name if the player location has a PlotGroup.
    - Closes #5539.
```
I opted to make a new placeholder, as the plotgroups are not altering the townblock names, (and probably shouldn't.) This will be a part of 0.97.5.3.
